using-mobile-extensions/adobe-analytics/README.md

```diff
@@ -4,7 +4,7 @@
 
 1. In Experience Platform Launch, click the **Extensions** tab.
 2. On the **Catalog** tab, locate the **Adobe Analytics** extension, and click **Install**.
-3. Type the extension settings. For more information, see [Configure the Analytics Extension](./#configure-analytics-extension).
+3. Type the extension settings. For more information, see [Configure the Analytics Extension](README.md#configure-analytics-extension).
```

Comment:
this link seems not to work - should be README.md#configure-the-analytics-extension

Reply:
Thanks @emdobrin, I didn't notice it was missing "the".  Fixed it.  All links  should work now.  Can you help to merge?